daml-assistant/BUILD.bazel

```diff
@@ -14,10 +14,12 @@ deps = [
     'filepath',
     'http-conduit',
     'main-tester',
+    'microlens',
```

Comment:
This is exactly one of the cases that I mentioned yesterday where you already have `lens` in your transitive dependency closure anyway (in this case, `da-hs-base` depends on it) so by adding `microlens` you potentially increase our dependency footprint instead of decreasing it.

Reply:
So I should just depend on `lens`?

Reply:
Either that or don’t use lenses.

Reply:
Sadly the SemVer library only exposes lenses 😬